internal/core/metadata/rest_device_test.go

```diff
@@ -19,17 +19,20 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	"github.com/pkg/errors"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	contract "github.com/edgexfoundry/go-mod-core-contracts/models"
+
 	"github.com/edgexfoundry/edgex-go/internal/core/metadata/interfaces"
 	"github.com/edgexfoundry/edgex-go/internal/core/metadata/interfaces/mocks"
 	"github.com/edgexfoundry/edgex-go/internal/pkg/config"
-	"github.com/edgexfoundry/go-mod-core-contracts/clients"
-	contract "github.com/edgexfoundry/go-mod-core-contracts/models"
-	"github.com/pkg/errors"
 )
 
 func TestGetAllDevices(t *testing.T) {
 	Configuration = &ConfigurationStruct{Service: config.ServiceInfo{MaxResultCount: 1}}
-	req, err := http.NewRequest(http.MethodGet, clients.ApiBase+"/"+DEVICE, nil)
+	_, err := http.NewRequest(http.MethodGet, clients.ApiBase+"/"+DEVICE, nil)
```

Comment:
@michaelestrin If we're not using the request anymore--do we still need to call `http.NewRequest` at all?

Reply:
I concur with @jdharms' assessment.  This (and the follow on `if` construct should be removed as they serve no purpose.  (Nice catch @jdharms!)